Source: Hong Hao <oahong@gmail.com>
Upstream: No
Reason: fix compilation with gcc-4.4
--- pcmanx-gtk2-0.3.8/src/nancy_bot/msgdata.cpp.orig	2009-05-20 19:42:37.447807417 +0800
+++ pcmanx-gtk2-0.3.8/src/nancy_bot/msgdata.cpp	2009-05-20 19:42:57.222554521 +0800
@@ -18,6 +18,7 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
+#include <stdio.h>
 #include <cstdlib>
 #include <unistd.h>
 
